src/dashboard/src/settings/base.py

```diff
@@ -438,7 +438,6 @@
 # Dashboard internal settings
 GEARMAN_SERVER = config.get("gearman_server")
 POLLING_INTERVAL = 5  # Seconds
-STATUS_POLLING_INTERVAL = 5  # Seconds
```

Comment:
I do notice in the network tab of my browser's dev tools that there seem to be two polling jobs:

![image](https://user-images.githubusercontent.com/6758804/98699674-63434280-2345-11eb-88e0-5f926faf9940.png)

`/transfer/status/?(timestamp)` is controlled by `POLLING_INTERVAL` and I've confirmed that changing that value is reflected in the Dashboard (the screenshot shows a polling interval of 16s, which is working nicely)

`/status/?(timestamp)` seems to happen every 5 seconds and responds with what I'm assuming is the number of transfers, SIPs, and DIPs awaiting decisions. Is it possible this `STATUS_POLLING_INTERVAL` is or was related to that?

Reply:
It's an interesting observation. The status that you noticed is controlled by some (unfortunately) hard-coded values, but this one specifically should be this one [here](https://github.com/artefactual/archivematica/blob/c5e7a0fb4aaaf5d30d20e46707b66f788d7a0fba/src/dashboard/src/media/js/status.js#L88). (Verified by changing on the fly). I don't think we know the original intent of `STATUS_POLLING_INTERVAL` but your hypothesis is as good as any. Douglas describes its non-use in the issue attached to the PR and I just confirmed that and removed it here.